src/components/molecules/contact-heading/contact-heading.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, VStack } from "@chakra-ui/react";
+import HeaderText from "components/atoms/typography/heading-text";
+import DefaultText from "components/atoms/typography/default-text";
+import Link from "components/atoms/link/link";
+import React from "react";
+
+const ContactHeading = () => {
+  return ( 
+    <Box as={"div"}>
+      <VStack justifyContent={"flex-start"} alignItems={"flex-start"}>
+        <HeaderText level={"h2"} className={"text-secondary py-2"}>
+                Contact Us
+        </HeaderText>
+        <DefaultText className={"text-gray"}>
+                You can reach us anytime via 
+          <br/>
+          <Link
+            url={"#"}>
+            <a href="mailto:chadstewart@techishiring.com" className={"text-primary"}>
+                      chadstewart@techishiring.com
```

Comment:
```suggestion
                      techishiring@gmail.com
```

Could you change this? Not your fault, completely forgot this needed to be changed in Figma, lol!